apps/api/src/migration/1696529645524-updatePermissionsEnum.ts

```diff
@@ -0,0 +1,45 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class updatePermissionsEnum1696529645524 implements MigrationInterface {
+  name = 'updatePermissionsEnum1696529645524';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TYPE "public"."allowed_activity_permission_enum" RENAME TO "allowed_activity_permission_enum_old"`,
```

Comment:
Nit - I have mixed feelings about having and old/new version of a column. Better practice would be to drop or update the column, and migrate the data as needed.